### ISSUE TITLE
How to visualize DurableEntity state in a liquid template

### ISSUE THREAD

**Issue by SamVanhoutte** [scale-tone/DurableFunctionsMonitor#51]
Hello ,

I wanted to create a liquid template that visualizes the state of a Durable Entity I have running in my solution.
I am able to write out the `InstanceId` in the template, but I don't seem to be able to access the State of the durable entity.

The following content is only rendering the `InstanceId` value, but none of the `RefreshCount` values that I have in the state of my entity.  It might be related to the fact that you indicate you are passing the DurableOrchestrationStatus object, while for Durable Entities, it may be required to have the DurableEntityStatus object passed, instead ?

```liquid
 <h2>Details for {{InstanceId}}</h2>

 <h3>Refresh count: {{Input.state.RefreshCount}}</h3>
 <h3>Refresh count: {{Input.State.RefreshCount}}</h3>
 <h3>Refresh count: {{Input.RefreshCount}}</h3>
 <h3>Refresh count: {{state.RefreshCount}}</h3>
 <h3>Refresh count: {{State.RefreshCount}}</h3>
 <h3>Refresh count: {{RefreshCount}}</h3>
```

Any hints or solution to this? 

Cheers and thanks for the awesome tool, again

**Comment by scale-tone** [comment]
Hi @SamVanhoutte , and may this New Year be better for you (for all of us, actually)! And thanks for your great feedback!

The Entity's state is indeed accessible via `Input.state` field. Sadly though, Durable Functions runtime serves that field in form of a *string* (and not as a JObject). 

This is a bug, of course. And it can be easily fixed on DFM's side, but you'll have to wait for next version.

Until then you could try to parse that string with Liquid tools, but it could be challenging... 
Tried to find any working Liquid filters, that would do the job of converting that string into an object - and no luck so far :(

**Comment by scale-tone** [comment]
fixed in v4.0

**Comment by SamVanhoutte** [comment]
Thanks for the update!  Just verified it and it works.  Appreciated
